ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.6.1f1
+m_EditorVersion: 2017.1.0f3
```

Comment:
Looked at it a bit, we should probably let Unity know what version the files are / not sure what it keeps track of & has to change between versions. Says pretty specifically on the website to keep the ProjectSettings folder